### PR TITLE
Add template for vnet workspace uses sdk v1 features only

### PR DESCRIPTION
**Pull request by mingweihe**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add template for vnet workspace uses sdk v1 features

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@Blackmist - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

**Comment by azure-quickstarts** [comment]
@Blackmist - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.machinelearningservices/machine-learning-workspace-vnet-sdk-v1/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Azure Machine Learning workspace (secure network configuration)
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/quickstarts/microsoft.machinelearningservices/machine-learning-workspace-vnet/PublicLastTestDate.svg)
```
[the remark this answers is not in the document]
Comment (mingweihe) [reply quickstarts/microsoft.machinelearningservices/machine-learning-workspace-vnet-sdk-v1/README.md]: > need to fix the links (details in the pipeline results)

learned, thanks

File: quickstarts/microsoft.machinelearningservices/machine-learning-workspace-vnet-sdk-v1/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+    "privateEndpointType": {
+      "type": "string",
+      "defaultValue": "none",
+      "allowedValues": [
+        "AutoApproval",
+        "ManualApproval",
+        "none"
+      ]
+    },
+	"encryption_cosmosdb_armid": {
```
[the remark this answers is not in the document]
Comment (mingweihe) [reply quickstarts/microsoft.machinelearningservices/machine-learning-workspace-vnet-sdk-v1/azuredeploy.json]: > format the json, e.g. SHIFT+ALT+F in VS Code

learned, thanks!

**Comment by azure-quickstarts** [comment]
@Blackmist - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@Blackmist - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by azure-quickstarts** [comment]
@Blackmist - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Comment by alex-frankel** [comment]
Merging since all tests are passing, but @bmoore-msft is OOF. He may take another look at this and request changes.

**Pull request merged**